src/utils/restUtilMod.F90.in

```diff
@@ -442,16 +447,34 @@ contains
        call ncd_putatt(ncid, varid, 'varnames_on_old_files', trim(varname))
 
        if (switchdim) then
-          status = PIO_put_att(ncid, vardesc%varid, 'switchdim_flag', 1)
+          status = PIO_put_att(ncid, vardesc%varid, 'switchdim_flag', iflag_switchdim)
        else
-          status = PIO_put_att(ncid, vardesc%varid, 'switchdim_flag', 0)
+          status = PIO_put_att(ncid, vardesc%varid, 'switchdim_flag', iflag_noswitchdim)
```

Comment:
`iflag_switchdim` and `iflag_noswitchdim` were declared but not used, so I used them here.